src/components/PerformanceQuestionsData/PerformanceQuestionsData.tsx

```
@@ -0,0 +1,336 @@
+  data: MaterialVariantData;
+}
+
+// --- Component types ---
+
+export enum PerformanceQuestionsVariant {
+  QUESTIONS = 'questions',
+  MATERIAL = 'material',
+}
+
+export interface PerformanceQuestionsDataProps
+  extends HTMLAttributes<HTMLDivElement> {
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)